src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingMethodSymbol.cs

```diff
@@ -359,5 +359,7 @@ internal override int CalculateLocalSyntaxOffset(int localPosition, SyntaxTree l
             // retargeting symbols refer to a symbol from another compilation, they don't define locals in the current compilation
             throw ExceptionUtilities.Unreachable;
         }
+
+        internal override bool IsNullableEnabled() => throw ExceptionUtilities.Unreachable;
```

Comment:
>throw ExceptionUtilities.Unreachable [](start = 54, length = 36)

I am curious what motivates the difference by comparison to PE? Both are "imported" symbols.